projects/demo/src/modules/app/classes/ts-file.parser.ts

```diff
@@ -0,0 +1,49 @@
+const CLASS_NAME_REGEXP = /(?<=export class\s)(\w*)/gi;
+
+export class TsFileParser {
+    get className(): string {
+        const [className] = this.rawFileContent.match(CLASS_NAME_REGEXP) || [''];
+
+        return className;
+    }
+
+    set className(newClassName: string) {
+        this.rawFileContent = this.rawFileContent.replace(
+            CLASS_NAME_REGEXP,
+            newClassName,
+        );
+    }
+
+    get isNgModule(): boolean {
+        return this.rawFileContent.includes('@NgModule');
+    }
+
+    get isNgComponent(): boolean {
```

Comment:
maybe rename `get isNgComponent` to `get hasNgComponents()`?

Reply:
done